src/components/Editor/Preview/Popup.js

```diff
@@ -318,15 +318,15 @@ export class Popup extends Component<Props, State> {
 }
 
 const mapStateToProps = state => ({
-  popupObjectProperties: getAllPopupObjectProperties(state),
-  openElementInInspector: actions.openElementInInspectorCommand
+  popupObjectProperties: getAllPopupObjectProperties(state)
 });
 
 const {
   addExpression,
   selectSourceURL,
   setPopupObjectProperties,
-  openLink
+  openLink,
+  openElementInInspector: actions.openElementInInspectorCommand
```

Comment:
this seems more reasonable from the outside 


```diff
diff --git a/src/components/Editor/Preview/Popup.js b/src/components/Editor/Preview/Popup.js
index 71412576f..039ccb955 100644
--- a/src/components/Editor/Preview/Popup.js
+++ b/src/components/Editor/Preview/Popup.js
@@ -325,15 +325,15 @@ const {
   addExpression,
   selectSourceURL,
   setPopupObjectProperties,
-  openLink,
-  openElementInInspector: actions.openElementInInspectorCommand
+  openLink
 } = actions;

 const mapDispatchToProps = {
   addExpression,
   selectSourceURL,
   setPopupObjectProperties,
-  openLink
+  openLink,
+  openElementInInspector: actions.openElementInInspectorCommand
 };

 export default connect(
```

Reply:
*sigh* bad copy/paste from MC.  Fixing now.

Reply:
de nada, feel free to open in phabricator as well